starter_code/src/movies.js

```diff
@@ -1,20 +1,157 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
+function turnHoursToMinutes (arr) {
```

Comment:
Se repite un poco el código.. os paso otra solución:

function turnHoursToMinutes(movies) {
  hoursToMin = movies.map(function(movie) {
    movieLength = movie.duration.split(" ");
    total = 0;

    for (var i = 0; i < movieLength.length; i++) {
      //console.log(movieLength[i])
      if (movieLength[i].includes("h")) {
        total += parseFloat(
          movieLength[i].slice(0, movieLength[i].indexOf("h")) * 60
        );
      } else {
        total += parseFloat(
          movieLength[i].slice(0, movieLength[i].indexOf("min"))
        );
      }
    }
    return { 
      duration: total
    };
  });
  return hoursToMin;
}